src/sentry/utils/sdk_crashes/cocoa_sdk_crash_detector.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any, Mapping, Sequence
+
+from sentry.utils.glob import glob_match
+from sentry.utils.sdk_crashes.sdk_crash_detector import SDKCrashDetector
+
+
+class CocoaSDKCrashDetector(SDKCrashDetector):
+    def __init__(self) -> None:
+        self
+
+    def is_sdk_crash(self, frames: Sequence[Mapping[str, Any]]) -> bool:
+        if not frames:
+            return False
+
+        # The frames are ordered from caller to callee, or oldest to youngest.
+        # The last frame is the one creating the exception.
+        # Therefore, we must iterate in reverse order.
+        for frame in reversed(frames):
+            if self.is_sdk_frame(frame):
+                return True
+
+            if frame.get("in_app") is True:
+                return False
+
+        return False
+
+    def is_sdk_frame(self, frame: Mapping[str, Any]) -> bool:
+        function = frame.get("function")
+
+        if function is not None:
+            # [SentrySDK crash] is a testing function causing a crash.
+            # Therefore, we don't want to mark it a as a SDK crash.
+            if "SentrySDK crash" in function:
+                return False
+
+            functionsMatchers = ["*sentrycrash*", "**[[]Sentry*"]
```

Comment:
python uses `snake_case`

Reply:
Thanks for pointing that out. Still getting used to Python. I'm a CamelCase 🐪 engineer.